servers/partner-ci/9.x/vcenter.yaml

```diff
@@ -81,14 +81,14 @@
     skip_vote_failed: false
 
     testrail_project: "Mirantis OpenStack"
-    testrail_suite: "[9.x][PCE] vCenter"
+    testrail_suite: "[9.1][PCE] vCenter"
     testrail_suite_desc: "Suite description"
     testrail_milestone: "9.2"
-    testrail_plan: "$TREP_TESTRAIL_MILESTONE snapshot #${{SNAPSHOTS_ID}}"
+    testrail_plan: "${{TREP_TESTRAIL_MILESTONE}} snapshot #${{SNAPSHOTS_ID}}"
```

Comment:
SNAPSHOTS_ID and TREP_TESTRAIL_MILESTONE are parameters and will be used as parameters.
Jenkins job parse them as bash variables before job run. 
Do you check it on jenkins?